homeassistant/components/tplink/manifest.json

```diff
@@ -169,5 +169,5 @@
   "iot_class": "local_polling",
   "loggers": ["kasa"],
   "quality_scale": "platinum",
-  "requirements": ["python-kasa[speedups]==0.5.4"]
```

Comment:
We likely have a lot more mac addresses to add to this since there are so many more supported devices now, but that can be another PR